lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyXSDURIResolver.java

```diff
@@ -71,7 +72,7 @@ public String resolve(String baseLocation, String publicId, String systemId) {
           //Check workspace for Liberty installation and generate schema.xsd file
           //Return schema URI as String, otherwise use cached schema.xsd file
           if (libertyWorkspace.isLibertyInstalled()) {
-            Path schemaGenJarPath = LibertyUtils.findFileInWorkspace(libertyWorkspace, "ws-schemagen.jar");
+            Path schemaGenJarPath = LibertyUtils.findFileInWorkspace(libertyWorkspace, Paths.get("bin/tools/ws-schemagen.jar"));
```

Comment:
I thought it was supposed to be more something like `Paths.get("bin", "tools", "ws-schemagen.jar")`?

Reply:
Fixed.